content/workshops/node/_index.md

```diff
@@ -1,11 +1,18 @@
 ---
-title: Node 
+title: Node
+ready: true
 ---
 
-## Introduction (slides)
+## Introduction
 
-- [Introduction to Node](https://docs.google.com/presentation/d/1dtrsWOWjoA9A9nTmznCkdYz0oRK6O76jXRINp0tzjrQ/edit?usp=sharing)
+The followings are to be discussed, demonstrated and addressed during the workshop
 
-- [Asynchronous vs Synchronous](https://docs.google.com/presentation/d/19ppsjmkKAvJcasK_BnjptTUfF4KQwaInDKVkhK1b6vA/edit?usp=sharing)
+1. What is Node.js
+2. Why Node js
+3. Node installation process
+4. NPM vs NPM
```

Comment:
NPM vs NPM doesn't make sense

Reply:
No topic exists yet for node.js at the moment

Reply:
Point number 4 doesn't make sense